infra/k8s/install.sh

```diff
@@ -66,19 +54,70 @@ while [ "$READY_NODES" -ne $(($WORKER_NODES + 1)) ]; do READY_NODES=$(kubectl ge
 echo "Cluster nodes are Ready"
 echo
 
-echo "Add secret for S3 bucket"
-echo
-kubectl create secret generic assets-s3-bucket --from-literal=access-key="$ASSETS_ACCESS_KEY" \
-                                               --from-literal=secret-key="$ASSETS_SECRET_KEY" \
-                                               --from-literal=s3-endpoint="$ASSETS_S3_ENDPOINT" \
-                                               --from-literal=bucket-name="$ASSETS_BUCKET_NAME"
+echo "Install EFS..."
 
+vpcId=`aws ec2 describe-vpcs --filters Name=tag:Name,Values=$NAME | jq ".Vpcs | .[] | .VpcId" | tr -d "\""`
+
+if [[ -z ${vpcId} ]]; then
+  echo "Couldn't detect AWS VPC created by `kops`"
+  exit 1
+fi
+
+echo "Detected VPC: $vpcId"
+
+securityGroupId=`aws ec2 describe-security-groups | jq ".SecurityGroups | .[] | select(.GroupName==\"nodes.$NAME\") | .GroupId" | tr -d '"'`
+
+if [[ -z ${securityGroupId} ]]; then
+  echo "Couldn't detect AWS Security Group created by `kops`"
+  exit 1
+fi
+
+echo "Detected Security Group ID: $securityGroupId"
+
+subnetId=`aws ec2 describe-subnets | jq ".Subnets | .[] | select (.VpcId == \"$vpcId\") | .SubnetId" | tr -d '"'`
+
+if [[ -z ${subnetId} ]]; then
+  echo "Couldn't detect AWS Subnet created by `kops`"
+  exit 1
+fi
+
+echo "Detected Subnet ID: $subnetId"
+
+pushd efs-terraform
+
+# extract s3 bucket from kops state store
+S3_BUCKET="${KOPS_STATE_STORE:5:100}"
+
+terraform init -backend-config=bucket=$S3_BUCKET \
+               -backend-config=key=tf-efs-$NAME \
+               -backend-config=region=$AWS_REGION
+
+terraform apply -var aws_region=$AWS_REGION -var fs_subnet_id=$subnetId -var fs_sg_id=$securityGroupId -auto-approve
+
+export EFS_DNSNAME=`terraform output dns_name`
+
+fsId=`terraform output filesystem_id`
+
+popd
+
+echo "Install EFS Kubernetes provisioner..."
+
+kubectl create configmap efs-provisioner \
+--from-literal=file.system.id=$fsId \
+--from-literal=aws.region=$AWS_REGION \
+--from-literal=provisioner.name=testground.io/aws-efs
+
+EFS_MANIFEST_SPEC=$(mktemp)
+envsubst <./efs/manifest.yaml.spec >$EFS_MANIFEST_SPEC
+
+kubectl apply -f ./efs/rbac.yaml \
+              -f $EFS_MANIFEST_SPEC
 
 echo "Install Weave, CNI-Genie, s3bucket DaemonSet, Sidecar Daemonset..."
 echo
 kubectl apply -f ./kops-weave/genie-plugin.yaml \
               -f ./kops-weave/weave.yml \
-              -f ./kops-weave/s3bucket.yml \
+              -f ./kops-weave/dummy.yml \
```

Comment:
We no longer need the `s3bucket` daemonset, but we still need at least one container on each host, so that the `cni0` interface is created on Flannel.